amqp/src/main/scala/akka/stream/alpakka/amqp/AmqpSinkStage.scala

```diff
@@ -9,22 +9,27 @@ import akka.util.ByteString
 import com.rabbitmq.client.AMQP.BasicProperties
 import com.rabbitmq.client._
 
-final case class OutgoingMessage(bytes: ByteString, immediate: Boolean, mandatory: Boolean, props: Option[BasicProperties])
+final case class OutgoingMessage(bytes: ByteString,
+                                 immediate: Boolean,
+                                 mandatory: Boolean,
+                                 props: Option[BasicProperties])
```

Comment:
It's (yet) impossible. Line width determines the behavior.

Guys, please let's not further discuss such small preferences. First, overall scalafmt enables much more uniform code than sclariform. Second, as you all know, it's impossible to arrive at a style which is preferred by all.